rest-api/model/questionnaire.py

```diff
@@ -15,6 +16,8 @@ class QuestionnaireBase(object):
   # The JSON representation of the questionnaire provided by the client.
   # Concepts and questions can be be parsed out of this for use in querying.
   resource = Column('resource', BLOB, nullable=False)
+  isValid = Column('is_valid', Enum(ValidQuestionnaireStatus),
```

Comment:
You could rename this to just "status" and maybe rename ValidQuestionnaireStatus to QuestionnaireDefinitionStatus I guess. I could imagine other statuses for questionnaires besides INVALID  / VALID.

Alternatively, just make it a Boolean "invalid" flag.